server/src/lib/utility.ts

```diff
@@ -8,3 +8,16 @@
 export function getFileExt(path: string): string {
   return path.substr(path.lastIndexOf('.') - path.length);
 }
+
+/**
+ * Returns the first defined argument. Returns null if there are no defined
+ * arguments.
+ */
+export function getFirstDefined(...options) {
+  for (var i = 0; i < arguments.length; i++) {
```

Comment:
nit: let's use `options` instead of `arguments` since you nicely used the spread operator above.